desktop/src/app.rs

```diff
@@ -341,13 +341,13 @@ impl App {
                                 _ => (),
                             }
 
-                            let key_code = winit_to_ruffle_key_code(&event);
+                            let key_code = winit_to_ruffle_key_code(dbg!(&event));
```

Comment:
Is this `dbg!` meant to be left in here?

Reply:
Whoops, those `dbg!`s do be sneaky!